.github/workflows/python-linters.yml

```diff
@@ -10,24 +12,53 @@ on:
 
 jobs:
   linters:
-    name: 🤖
+    name: >-
+      ${{ matrix.env.TOXENV }}
+      /
+      ${{ matrix.python-version }}
+      /
+      ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       # max-parallel: 5
       matrix:
+        python-version:
+        - 3.8
         os:
-        - ubuntu-18.04
+        - ubuntu-latest
         - windows-latest
         - macos-latest
         env:
         - TOXENV: docs
         - TOXENV: lint
+        - TOXENV: vendoring
+
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
+
     steps:
     - uses: actions/checkout@master
-    - name: Set up Python ${{ matrix.env.PYTHON_VERSION || 3.8 }}
+    - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v1
       with:
-        python-version: ${{ matrix.env.PYTHON_VERSION || 3.8 }}
+        python-version: ${{ matrix.python-version }}
+    - name: Log Python version
+      run: >-
+        python --version
+    - name: Log Python location
+      run: >-
+        which python
+    - name: Log Python env
+      run: >-
+        python -m sysconfig
```

Comment:
nit: I'd prefer to merge these into one. Reasons detailed in a later comment.

Reply:
(explained @ https://github.com/pypa/pip/pull/7611/files#r368251546) It's easier to inspect/separate outputs of different commands if each has a separate label. I think that for me it's coming from Ansible's best practices of always naming steps...